contracts/dao/Getters.sol

```diff
@@ -95,10 +95,41 @@ contract Getters is State {
         return _state.balance.coupons;
     }
 
-    function totalNet() public view returns (uint256) {
-        return dollar().totalSupply().sub(totalDebt());
+    function treasury() public view returns (address) {
+        return Constants.getTreasuryAddress();
     }
 
+    // DIP-10
+    function totalCDSDBonded() public view returns (uint256) {
+        return cdsd().balanceOf(address(this));
+    }
+
+    function dip10TotalRedeemable() public view returns (uint256) {
+        return _state10.dip10TotalRedeemable;
+    }
+
+    function globalInterestMultiplier() public view returns (uint256) {
+        return _state10.globalInterestMultiplier;
+    }
+
+    function expansionStartEpoch() public view returns (uint256) {
+        return _state10.expansionStartEpoch;
+    }
+
+    function dip10TotalRedeemed() public view returns (uint256) {
+        return _state10.dip10TotalRedeemable;
```

Comment:
Shouldn't be `_state10.dip10TotalRedeemed`?

Reply:
Good point! @devnulldsd , it was added in the last commit, any reasoning for this?

Reply:
It looks incorrect. Thanks for pointing out @acatinon.